### ISSUE TITLE
Error when running slidify for the first time

### ISSUE THREAD

**Issue by pdiazs** [ramnathv/slidify#373]
I get this error 
Windows 7 64b
Rstudio
Install as demanded

slidify("index.Rmd")
processing file: index.Rmd
  |.................................................................| 100%
  ordinary text without R code
output file: index.md
Error in yaml.load(gsub("^---\n+", "", txt[1])) : 
  Reader error: invalid trailing UTF-8 octet: #61 at 101

**Comment by Saoirsegirl** [comment]
I am experiencing the same issue.  I have verified the encoding is utf-8; run Ramnath's test-deck and it fails with the error.

Error in if (body$content == "") { : 
  missing value where TRUE/FALSE needed

The first time, I made no alterations to the template created from the author() command.
The second time I added simple entries for the first 4 YAML statements. (they line up in RStudio
title       : test
subtitle    : match Ramnath
author      : Barb
job         : confused

**Comment by mdcscry** [comment]
install_github('ramnathv/slidify', ref = github_pull(425))

 https://github.com/ramnathv/slidify/issues/407

**Comment by shahronak47** [comment]
Thank you @mdcscry that resolved my issue!

**Comment by bruceloco** [comment]
Yeah, it is just sad that I have to waste time writing projects for coursera and have to hit my head against the walls because the software versions are not stable/tested

**Comment by ivanko144** [comment]
Thank you @mdcscry that resolved my issue!

**Comment by anirbanc81** [comment]
Thanks! @mdcscry fixed my problem too....just wondering if you or anyone from the team could let us know what the issue was? Cheers!

**Comment by mdcscry** [comment]
Anibanc81--if you click that link #407 above you can see the author had some regex issues that he fixed right up.  Another awesome netizen.

**Comment by smidge84** [comment]
Thanks @mdcscry, your update fixed my problem too. Much appreciated.
I agree with @bruceloco, about trying to do Coursera project and hitting loads of brick walls regarding to the software. I never got the publishing to rPubs to work. Currently in the Developing Data Products course I have written my shiny app but get dll file errors and other stuff when trying to deploy it to shinyapps.io.
This is wasting what limited time I have to do the assignments in the first place, creating additional difficulties/stress in my life when I should also be trying to relax in my free time, not consumer more than is necessary by Coursera.

**Comment by kstseng** [comment]
Thanks to @mdcscry !! Problem fixed!!
